backend/danswer/server/admin.py

```diff
@@ -35,7 +35,7 @@ def modify_slack_config(slack_config: SlackConfig):
 
 
 @router.post("/connectors/slack/auth")
```

Comment:
this should probably just get removed? not sure what it's supposed to do

Reply:
broski you wrote this one

Reply:
removed!